### DISCUSSION TITLE
Sliderends get unsnapped after changing slider anchor type

### DISCUSSION THREAD

**Discussion by cihe13375** [ppy/osu#29903] [Q&A] [answered]
1. create a linear slider
2. change the first anchor to perfect curve
3. now the sliderend is unsnapped

it also happens in the reverse way, i.e. create a perfect curve first and then change to linear.

I once thought this is a feature but decide to report it after seeing someone with 99999 sliderends unsnapped in the verify tab and *I guess* this is the most likely reason

https://github.com/user-attachments/assets/8c49a821-e35e-4b96-ac22-146754c4aefe

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/29915
